src/App.jsx

```diff
@@ -15,6 +15,7 @@ const App = () => {
   const [reviewsTotal, setReviewsTotal] = useState(0);
   const [isLoaded, setIsLoaded] = useState(false);
   const [isError, setIsError] = useState(false);
+  const [reviewMetaObj, setMeta] = useState({});
```

Comment:
The set state should always follow same name as the state, so setReviewMetaObj. I also have in the branch I'm working on state for reviewMetaInfo and stylesInfo to match the productInfo naming, so we should pick one and stay consistent.